### PR TITLE
GEOMESA-692 Remove Duplicated Interface

### PR DESCRIPTION
**Pull request by AndrewAnnex**
* AccumuloBackedRasterOperations has now been replaced by the RasterStore

### CODE REVIEW

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala [hunk 1]
```diff
@@ -17,50 +17,267 @@
...
+  }
+
+  def getAvailableResolutions(): Seq[Double] = {
+    // TODO: Consider adding resolutions + extent info  https://geomesa.atlassian.net/browse/GEOMESA-645
+    getAvailabilityMap().keySet.toSeq.sorted
+  }
 
-  def ensureBoundsTableExists() = rasterOps.ensureBoundsTableExists()
+  def getAvailableGeoHashLengths(): Seq[Int] = {
+    getAvailabilityMap().values.toSeq.distinct
```
Comment (apc3n) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala]: why not return a Set[Int]?

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala [hunk 2]
```diff
@@ -17,50 +17,267 @@
...
+  }
 
-  def getVisibility = rasterOps.getVisibility()
+  def getGridRange(): GridEnvelope2D = {
+    val bounds = getBounds()
+    val resolutions = getAvailableResolutions()
+    // If no resolutions are available, then we have an empty table so assume 1.0 for now
+    val resolution = resolutions match {
+      case Nil => 1.0
+      case _   => resolutions.min
```
Comment (apc3n) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala]: match on List(_) or something this case _ match could match on null or some other nonsense

Comment (jnh5y) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala]: resolutions is a Seq, so it is empty or has something.  This is typesafe.

File: geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala [hunk 3]
```diff
@@ -17,50 +17,267 @@
...
+  def putRaster(raster: Raster): Unit
+  def getBounds(): BoundingBox
+  def getAvailableResolutions(): Seq[Double]
+  def getAvailableGeoHashLengths(): Seq[Int]
+  def getAvailabilityMap(): ImmutableSetMultimap[Double, Int]
+  def getGridRange(): GridEnvelope2D
+  def getMosaicedRaster(query: RasterQuery, params: GeoMesaCoverageQueryParams): BufferedImage
+}
+
+class RasterStore(val connector: Connector,
```
Comment (jnh5y) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala]: Do we want to call this an AccumuloRasterStore or something else to indicate the backing store?

Comment (AndrewAnnex) [inline geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala]: sure can

**Comment by jnh5y** [comment]
Committed as e4fab6b9bcd4bc00b09d0871111023a415cda1a2 on f_wcs_4.

**Pull request closed without merging**